profiler/src/ProfilerEngine/Datadog.Profiler.Native/Configuration.h

```diff
@@ -76,6 +76,7 @@ class Configuration final : public IConfiguration
     bool _isExceptionProfilingEnabled;
     bool _isAllocationProfilingEnabled;
     bool _isContentionProfilingEnabled;
+    bool _isEndpointProfilingEnabled;
```

Comment:
Is this being used anywhere?

Reply:
Removed in https://github.com/DataDog/dd-trace-dotnet/pull/3015/commits/520872d8381d8947b26576f1ca8e25441d3627d7